docs/try-leverage/add-aws-accounts.md

```diff
@@ -0,0 +1,350 @@
+# Add more AWS Accounts
+
+## Brief
+You can add new AWS accounts to your Leverage project by following the steps in this page.
+
+!!! info "Important"
+    In the examples belo, we will be using `apps-prd` as the account we will be adding and it will be created in the `us-east-1` region.
```

Comment:
```suggestion
    In the examples below, we will be using `apps-prd` as the account we will be adding and it will be created in the `us-east-1` region.
```